docker/arches_her/docker-compose.yml

```diff
@@ -0,0 +1,92 @@
+version: '3.8'
+services:
+  archesher:
+    container_name: archesher
+    image: he/arches_her:dev_build
```

Comment:
@RobOatesHistoricEngland - can we change the reference to `he` in the  image name `he/arches_her:dev_build` to `aher`.

Reply:
@aj-he Like this `aher/arches_her:dev_build`?